flutter/lib/src/feedback/sentry_feedback_widget.dart

```diff
@@ -0,0 +1,242 @@
+                      ),
+                      keyboardType: TextInputType.emailAddress,
+                      validator: (String? value) {
+                        return _errorText(value, widget.isEmailRequired);
+                      },
+                      autovalidateMode: AutovalidateMode.onUserInteraction,
+                    ),
+                    const SizedBox(height: 16),
+                    Row(
+                      children: [
+                        Text(
+                          key: const ValueKey('sentry_feedback_message_label'),
+                          widget.messageLabel,
+                          style: Theme.of(context).textTheme.labelMedium,
+                        ),
+                        const SizedBox(width: 4),
+                        Text(
+                          key: const ValueKey(
+                              'sentry_feedback_message_required_label'),
+                          widget.isRequiredLabel,
+                          style: Theme.of(context).textTheme.labelMedium,
+                        ),
+                      ],
+                    ),
+                    const SizedBox(height: 4),
+                    TextFormField(
+                      key: const ValueKey('sentry_feedback_message_textfield'),
+                      controller: _messageController,
+                      style: Theme.of(context).textTheme.bodyLarge,
+                      minLines: 5,
+                      maxLines: null,
+                      decoration: InputDecoration(
+                        border: const OutlineInputBorder(),
+                        hintText: widget.messagePlaceholder,
+                      ),
+                      keyboardType: TextInputType.multiline,
+                      validator: (String? value) {
+                        return _errorText(value, true);
+                      },
+                      autovalidateMode: AutovalidateMode.onUserInteraction,
+                    ),
+                  ],
+                ),
+              ),
+            ),
+            Column(
+              children: [
+                SizedBox(
+                  width: double.infinity,
+                  child: ElevatedButton(
+                    key: const ValueKey('sentry_feedback_submit_button'),
+                    onPressed: () async {
+                      if (!_formKey.currentState!.validate()) {
+                        return;
+                      }
+                      final feedback = SentryFeedback(
+                        message: _messageController.text,
+                        contactEmail: _emailController.text,
+                        name: _nameController.text,
+                        associatedEventId: widget.associatedEventId,
+                      );
+                      await _captureFeedback(feedback);
+                      // ignore: use_build_context_synchronously
+                      Navigator.pop(context);
```

Comment:
This will lead to weird behavior when the user uses `Navigator.pushReplacement()` to push this page from the user's root page.

Using `maybePop` fixes that.
```suggestion
                      Navigator.maybePop(context);
```

Not sure if or how often that is a problem in practice, though.